tools/dotcms-cli/api-data-model/src/main/java/com/dotcms/api/client/model/RemoteURLParam.java

```diff
@@ -0,0 +1,18 @@
+package com.dotcms.api.client.model;
+
+import java.net.URL;
+import java.util.Optional;
+
+/**
+ * The RemoteURLParam interface is used in the {@link com.dotcms.cli.common.AuthenticationMixin} to
+ * set and get the dotCMS URL parameter. It provides methods to set and retrieve the URL.
+ */
+public interface RemoteURLParam {
```

Comment:
it would be great to have interface such as Resetable

Reply:
Done, `Resetable` interface created, thanks for the feedback!